bors.toml

```diff
@@ -1,5 +1,4 @@
 status = [
   "continuous-integration/travis-ci/push",
-  # "continuous-integration/appveyor/branch"
```

Comment:
why this?

Reply:
just a drive-by edit: we've removed appveyor long ago. If we reinstall windows CI, that would be via azure pipelines. Though, ti makes total sense to send this as a separate change

Reply:
i understand